source/nodejs/adaptivecards-designer/index.html

```diff
@@ -18,7 +18,9 @@
 
 	<script type="text/javascript">
 		window.onload = function() {
-			if (!ACDesigner.SettingsManager.isLocalStorageAvailable) {
+            ACDesigner.GlobalSettings.previewFeaturesEnabled = true;
```

Comment:
>ACDesigner.GlobalSettings.previewFeaturesEnabled = true; [](start = 12, length = 56)

Do we want this to be enabled by default in masteR?

Reply:
Good catch - no, we don't want that. I turned it off.